R/frgtracker.R

```diff
@@ -1,4 +1,34 @@
 
+grades <- data.frame(
```

Comment:
Yanhua, you are not supposed to submit the `grades` and `courses` objects in the main code.